src/experimental/types.d.ts

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
```

Comment:
👍 Good idea with moving all the experiment related type definitions into these separate files.